nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -351,9 +373,29 @@ contract StakingEscrow is Issuer {
         return workerToStaker[_worker];
     }
 
+    /**
+    * @notice Get work that done by the staker
+    **/
+    function getWorkDone(address _staker) public view returns (uint256) {
```

Comment:
This function name reads more like a verb than a getter.

Reply:
✔️